Library/Homebrew/requirements/x11_requirement.rb

```diff
@@ -17,32 +13,26 @@ def initialize(name = "x11", tags = [])
   end
 
   def min_version
-    # TODO: remove in https://github.com/Homebrew/brew/pull/3483
-    return Version::NULL unless OS.mac?
-
-    MacOS::XQuartz.minimum_version
+    Version.new "1.12.2"
```

Comment:
Don't think this actually needs a `Version.new` given you're always comparing them to `Version`s below.